bernoulli.cpp(46): error C2653: 'Reduction': is not a class o r namespace name...

Using windows with VS 15 2017, using the commands in the readme. When I build with cmake I get errors on the bernoulli.cpp.

```
C:\Users\jespe\Documents\pytorch-cpp-rl\src\distributions\bernoulli.cpp(46): error C2653: 'Reduction': is not a class o
r namespace name [C:\Users\jespe\Documents\pytorch-cpp-rl\build\cpprl.vcxproj]
C:\Users\jespe\Documents\pytorch-cpp-rl\src\distributions\bernoulli.cpp(46): error C2065: 'None': undeclared identifier
 [C:\Users\jespe\Documents\pytorch-cpp-rl\build\cpprl.vcxproj]
C:\Users\jespe\Documents\pytorch-cpp-rl\src\distributions\bernoulli.cpp(52): error C2653: 'Reduction': is not a class o
r namespace name [C:\Users\jespe\Documents\pytorch-cpp-rl\build\cpprl.vcxproj]
C:\Users\jespe\Documents\pytorch-cpp-rl\src\distributions\bernoulli.cpp(52): error C2065: 'None': undeclared identifier
 [C:\Users\jespe\Documents\pytorch-cpp-rl\build\cpprl.vcxproj]
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -20,7 +20,7 @@ before_install:
   - wget -O cmake.sh https://cmake.org/files/v3.10/cmake-3.10.0-rc1-Linux-x86_64.sh
   - sudo sh cmake.sh --skip-license --exclude-subdir --prefix=/usr/local
   # Install libtorch
-  - wget https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.4.0.zip -O /tmp/libtorch.zip
+  - wget https://download.pytorch.org/libtorch/cpu/libtorch-shared-with-deps-1.4.0%2Bcpu.zip -O /tmp/libtorch.zip
   - unzip /tmp/libtorch.zip -d /tmp/
   # Install cppcheck
   - git clone https://github.com/danmar/cppcheck /tmp/cppcheck && cd /tmp/cppcheck && git checkout 1.87 && cd -

--- a/.travis.yml
+++ b/.travis.yml
@@ -20,7 +20,7 @@ before_install:
   - wget -O cmake.sh https://cmake.org/files/v3.10/cmake-3.10.0-rc1-Linux-x86_64.sh
   - sudo sh cmake.sh --skip-license --exclude-subdir --prefix=/usr/local
   # Install libtorch
-  - wget https://download.pytorch.org/libtorch/cpu/libtorch-shared-with-deps-1.3.0%2Bcpu.zip -O /tmp/libtorch.zip
+  - wget https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.4.0.zip -O /tmp/libtorch.zip
   - unzip /tmp/libtorch.zip -d /tmp/
   # Install cppcheck
   - git clone https://github.com/danmar/cppcheck /tmp/cppcheck && cd /tmp/cppcheck && git checkout 1.87 && cd -

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -13,7 +13,7 @@ before_build:
   - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
   - git submodule update --init --recursive
   # Install libtorch
-  - curl -fsS -o libtorch.zip https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.4.0.zip
+  - curl -fsS -o libtorch.zip https://download.pytorch.org/libtorch/cpu/libtorch-shared-with-deps-1.4.0%2Bcpu.zip
   - 7z x libtorch.zip -y -oC:\
   - set PATH=C:\libtorch\lib;%PATH%
   # Run CMake

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -13,7 +13,7 @@ before_build:
   - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
   - git submodule update --init --recursive
   # Install libtorch
-  - curl -fsS -o libtorch.zip https://download.pytorch.org/libtorch/cpu/libtorch-shared-with-deps-1.4.0%2Bcpu.zip
+  - curl -fsS -o libtorch.zip https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.4.0.zip
   - 7z x libtorch.zip -y -oC:\
   - set PATH=C:\libtorch\lib;%PATH%
   # Run CMake

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -13,7 +13,7 @@ before_build:
   - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
   - git submodule update --init --recursive
   # Install libtorch
-  - curl -fsS -o libtorch.zip https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.3.0.zip
+  - curl -fsS -o libtorch.zip https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.4.0.zip
   - 7z x libtorch.zip -y -oC:\
   - set PATH=C:\libtorch\lib;%PATH%
   # Run CMake

--- a/src/distributions/bernoulli.cpp
+++ b/src/distributions/bernoulli.cpp
@@ -43,13 +43,13 @@ Bernoulli::Bernoulli(const torch::Tensor *probs,
 
 torch::Tensor Bernoulli::entropy()
 {
-    return torch::binary_cross_entropy_with_logits(logits, probs, torch::Tensor(), torch::Tensor(), Reduction::None);
+    return torch::binary_cross_entropy_with_logits(logits, probs, torch::Tensor(), torch::Tensor(), torch::Reduction::None);
 }
 
 torch::Tensor Bernoulli::log_prob(torch::Tensor value)
 {
     auto broadcasted_tensors = torch::broadcast_tensors({logits, value});
-    return -torch::binary_cross_entropy_with_logits(broadcasted_tensors[0], broadcasted_tensors[1], torch::Tensor(), torch::Tensor(), Reduction::None);
+    return -torch::binary_cross_entropy_with_logits(broadcasted_tensors[0], broadcasted_tensors[1], torch::Tensor(), torch::Tensor(), torch::Reduction::None);
 }
 
 torch::Tensor Bernoulli::sample(c10::ArrayRef<int64_t> sample_shape)

